src/agent/protocols/protos/agent.proto

```diff
@@ -11,8 +11,8 @@ option go_package = "github.com/kata-containers/kata-containers/src/runtime/virt
 
 package grpc;
 
-import "github.com/kata-containers/kata-containers/src/agent/protocols/protos/oci.proto";
-import "github.com/kata-containers/kata-containers/src/agent/protocols/protos/github.com/kata-containers/agent/pkg/types/types.proto";
+import "oci.proto";
+import "github.com/kata-containers/agent/pkg/types/types.proto";
```

Comment:
Please use the one in `kata-containers/kata-containers` repository instead of the go agent repository.

Reply:
@bergwolf I used the relative path and have just moved the `types.proto` out of that folder now.